app/views/cases/sar_internal_review/_new_form_common.html.slim

```diff
@@ -0,0 +1,65 @@
+section
+  header
```

Comment:
How come we're using header here? and further down?

Reply:
Just an oversight no real reason.

Reply:
Removed 👍